tfhe/src/integer/gpu/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -717,4 +727,145 @@ mod tests {
             }
         }
     }
+
+    #[test]
+    fn test_gpu_ciphertext_compression_fast_path() {
+        // these parameters are insecure
+        const PARAM_CUSTOM_FAST_PATH: ClassicPBSParameters = ClassicPBSParameters {
```

Comment:
@IceTDrinker , @agnesLeroy here I added dummy parameters that use the fast path but have no correctness guarantee - thus I removed the `and` operation in the test so no PBS is done -> just encrypt/compress/extract/decrypt. 

I hope this is ok, let me know.